### PR TITLE
feat: add user-agent logging to acks

### PR DESCRIPTION
**Pull request by jrconlin**
Closes: #349 

@bbangert r?

### CODE REVIEW

**Comment by bbangert** [comment]
Type should be 'feat' not 'feature' per CONTRIBUTING.

File: autopush/websocket.py [hunk 1]
```diff
@@ -1090,7 +1094,8 @@ def ver_filter(update):
             size = len(msg.data) if msg.data else 0
             log.msg("Ack", router_key="webpush", channelID=chid,
                     message_id=version, message_type="direct",
-                    message_size=size, uaid_hash=self.ps.uaid_hash)
+                    message_size=size, uaid_hash=self.ps.uaid_hash,
+                    **{'user-agent': self.ps.user_agent})
```
Comment (bbangert) [inline autopush/websocket.py]: All the logging keys here are using underscore, why not just user_agent=... and skip the ** dict?

Comment (jrconlin) [inline autopush/websocket.py]: Ah, I thought you wanted this to match how we're logging in endpoint. There, it's going in as "user-agent" thanks to pretty much the same thing via client_info.

Perfectly happy to use user_agent if that's not a problem with the logging stuff.

Comment (bbangert) [inline autopush/websocket.py]: Oh, my bad, ugh, we use underscore and dash randomly. How annoying. Let's file a separate bug to have a code-wide consistency to all logging keys cause this is getting out of hand already.

File: autopush/websocket.py [hunk 2]
```diff
@@ -1090,7 +1094,8 @@ def ver_filter(update):
             size = len(msg.data) if msg.data else 0
             log.msg("Ack", router_key="webpush", channelID=chid,
                     message_id=version, message_type="direct",
-                    message_size=size, uaid_hash=self.ps.uaid_hash)
+                    message_size=size, uaid_hash=self.ps.uaid_hash,
+                    userr_agent=self.ps.user_agent)
```
Comment (bbangert) [inline autopush/websocket.py]: typo: userrrr

**Comment by bbangert** [comment]
r+

**Pull request merged**